Doc/library/typing.rst

```diff
@@ -3024,7 +3024,9 @@ Introspection helpers
 
    This is often the same as ``obj.__annotations__``. In addition,
    forward references encoded as string literals are handled by evaluating
-   them in ``globals`` and ``locals`` namespaces. For a class ``C``, return
+   them in ``globals``, ``locals`` and (where applicable)
+   :ref:`type parameter <annotation-scopes>` namespaces.
```

Comment:
This should link to https://docs.python.org/3.12/reference/compound_stmts.html#type-params instead. Annotation scopes are a more general concept.
```suggestion
   :ref:`type parameter <type-params>` namespaces.
```